<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowResultDialog</class>
 <widget class="QDialog" name="ShowResultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>18</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>現在画像</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetResult">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>371</width>
     <height>401</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Comment</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>適合率</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>分類</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>661</width>
     <height>18</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>登録画像</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegOK">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK登録</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCurrentComment">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegNG">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>NG登録</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadRegPack">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>読込</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveRegPack">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClearRegPack">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>470</y>
     <width>101</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>クリア</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>470</y>
     <width>121</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>321</width>
     <height>18</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>登録リスト</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>281</width>
     <height>411</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QPushButton" name="pushButtonDeleteReg">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>削除</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Comment</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeReg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>変更</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>分類</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRegComment">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxJudge">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>OK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NG</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelRegPanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelCurrentPanel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
